src/app/public/src/modules/shared/omnibar-adapter.service.ts

```diff
@@ -5,8 +5,10 @@ import { StacheWindowRef } from './window-ref';
 @Injectable()
 export class StacheOmnibarAdapterService {
   private static readonly HAS_OMNIBAR_CLASS_NAME: string = 'stache-omnibar-enabled';
+  private static readonly EXPECTED_OMNIBAR_MAX_HEIGHT: number = 100;
```

Comment:
Why `100`, instead of `50`?

Reply:
I wanted a bit of a buffer just in case for some reason the `Omnibar` changes height by a few pixels.  Otherwise we could just not bother detecting the bar and just entering the static height.